src/internal/connector/exchange/restore_test.go

```diff
@@ -217,36 +217,38 @@ func (suite *RestoreIntgSuite) TestRestoreExchangeObject() {
 				return ptr.Val(folder.GetId())
 			},
 		},
-		{
-			name: "Test Mail: Hydrated Item Attachment Mail",
-			bytes: exchMock.MessageWithNestedItemAttachmentMail(t,
-				exchMock.MessageBytes("Basic Item Attachment"),
-				"Mail Item Attachment",
-			),
-			category: path.EmailCategory,
-			destination: func(t *testing.T, ctx context.Context) string {
-				folderName := tester.DefaultTestRestoreDestination("mailbasicattch").ContainerName
-				folder, err := suite.ac.Mail().CreateMailFolder(ctx, userID, folderName)
-				require.NoError(t, err, clues.ToCore(err))
 
-				return ptr.Val(folder.GetId())
-			},
-		},
-		{
-			name: "Test Mail: Hydrated Item Attachment Mail One Attach",
-			bytes: exchMock.MessageWithNestedItemAttachmentMail(t,
-				exchMock.MessageWithDirectAttachment("Item Attachment Included"),
-				"Mail Item Attachment",
-			),
-			category: path.EmailCategory,
-			destination: func(t *testing.T, ctx context.Context) string {
-				folderName := tester.DefaultTestRestoreDestination("mailnestattch").ContainerName
-				folder, err := suite.ac.Mail().CreateMailFolder(ctx, userID, folderName)
-				require.NoError(t, err, clues.ToCore(err))
-
-				return ptr.Val(folder.GetId())
-			},
-		},
+		//TODO: Neha complete as part of https://github.com/alcionai/corso/issues/2428
```

Comment:
is this PR still a work in process, then?  If so, you should move it to `draft`.

Reply:
No. Actually we don't handle the nested attachment as of now. I will take that issue next. Till then we need to comment out the nested attachment test cases.  
I added a todo to make sure the test cases are taken care of in the next issue.